src/components/HelpSidebarIcons.tsx

```diff
@@ -59,16 +63,46 @@ export function HelpSidebarIcons(props: Props) {
   const updateCompanyUser = useUpdateCompanyUser();
   const handleUserChange = useHandleCurrentUserChangeProperty();
 
+  const { data: latestVersion } = useQuery({
+    queryKey: ['/pdf.invoicing.co/api/version'],
+    queryFn: () =>
+      axios
+        .get('https://pdf.invoicing.co/api/version')
+        .then((response) => response.data),
+    staleTime: Infinity,
+  });
+
+  const { data: currentSystemInfo } = useQuery({
+    queryKey: ['/api/v1/health_check'],
+    queryFn: () =>
+      request('GET', endpoint('/api/v1/health_check')).then(
+        (response) => response.data
+      ),
+    staleTime: Infinity,
+    enabled:
+      isSelfHosted() &&
+      !(import.meta.env.VITE_API_URL as string).includes('staging'), // Note: The staging API helped me test this functionality, but the health_check endpoint is not available on the staging API.
```

Comment:
@beganovich Hmm, I don't think so. This constraint was placed here because this endpoint is not available on staging, and because of that, we are getting unnecessary errors that need to be handled somehow. So, basically, this should stay because of the staging environment.

Reply:
@turbo124 leaving this PR open until we sync staging, don't think it's smart merging such a code :+1: